.github/workflows/build-in-devcontainer.yaml

```diff
@@ -88,18 +89,23 @@ jobs:
             AWS_SECRET_ACCESS_KEY=${{ env.AWS_SECRET_ACCESS_KEY }}
           runCmd: |
             set -e;
-
             mkdir -p ~/.config/pip/;
             cat <<EOF >> ~/.config/pip/pip.conf
             [global]
             extra-index-url = https://pypi.anaconda.org/rapidsai-wheels-nightly/simple
             EOF
 
-            rapids-make-${PYTHON_PACKAGE_MANAGER}-env || true;
+            rapids-make-${PYTHON_PACKAGE_MANAGER}-env;
```

Comment:
Was this changed because you want the job to fail if environment creation does?

Reply:
Yes